src/main/java/com/revature/RevRelay/utils/JwtUtil.java

```diff
@@ -27,17 +24,23 @@
     //this will obviously need to not be in the source code in the project deliverable.
 //    @Value("${jwt.secret}")
```

Comment:
Someone who understands what's happening in this file, please implement this instead.